app/models/agent.rb

```diff
@@ -466,4 +466,12 @@ def type
       @object.__send__(attr)
     } unless method_defined?(attr)
   }
+
+  def working
```

Comment:
I assume `working?` does not work well in Liquid?

Reply:
I don't think `?` can be a part of an identifier in Liquid. 😄